Moderation-Policy.md

```diff
@@ -0,0 +1,48 @@
+## Moderation policy
+
+### Applicability
+
+By default, this policy applies to all repositories under the Node.js Github Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects chartered by the TSC may adopt an alternative Moderation Policy for any repository under their stewardship so long as:
```

Comment:
Good question. That's an edge case that's not really addressed anywhere in the governance documents. Perhaps we need to be able to say that every repo has a primary owner which it either the TSC, CTC or a WG.